images/builder/Dockerfile

```diff
@@ -17,43 +17,47 @@
 # If you make changes to this Dockerfile run `make builder-image-push`.
 
 # Install Lychee
-FROM docker.io/library/alpine:3.19.1@sha256:6457d53fb065d6f250e1504b9bc42d5b6c65941d57532c072d929dd0628977d0 as lychee
-# update: datasource=github-tags depName=lycheeverse/lychee versioning=semver
-ENV LYCHEE_VERSION="v0.14.3"
-# hadolint ignore=DL3018
-RUN apk add --no-cache curl && \
-    curl -L -o /tmp/lychee-${LYCHEE_VERSION}.tgz https://github.com/lycheeverse/lychee/releases/download/${LYCHEE_VERSION}/lychee-${LYCHEE_VERSION}-x86_64-unknown-linux-gnu.tar.gz && \
-    tar -xz -C /tmp -f /tmp/lychee-${LYCHEE_VERSION}.tgz && \
-    mv /tmp/lychee /usr/bin/lychee && \
-    rm -rf /tmp/linux-amd64 /tmp/lychee-${LYCHEE_VERSION}.tgz
+FROM docker.io/lycheeverse/lychee:0.14.3 as lychee
+# Install Hadolint
+FROM docker.io/hadolint/hadolint:v2.12.0-alpine@sha256:7dba9a9f1a0350f6d021fb2f6f88900998a4fb0aaf8e4330aa8c38544f04db42 as hadolint
 
-# Install Golang CI Lint
-FROM docker.io/library/alpine:3.19.1@sha256:6457d53fb065d6f250e1504b9bc42d5b6c65941d57532c072d929dd0628977d0 as golangci
-# update: datasource=github-tags depName=golangci/golangci-lint versioning=semver
+FROM cgr.dev/chainguard/wolfi-base:latest as wolfi
+
+# update: datasource=github-tags depName=kubernetes-sigs/cluster-api
+ARG CLUSTERCTL_VERSION="v1.6.3"
+# update: datasource=github-tags depName=golangci/golangci-lint
 ENV GOLANGCI_VERSION="v1.56.2"
-WORKDIR /
-# hadolint ignore=DL3018,DL4006
-RUN apk add --no-cache curl && \
-    curl -L -o /tmp/golangci.tgz https://github.com/golangci/golangci-lint/releases/download/${GOLANGCI_VERSION}/golangci-lint-${GOLANGCI_VERSION#v}-linux-amd64.tar.gz && \
-    ls -la /tmp && \
-    tar -xz -C /tmp -f /tmp/golangci.tgz && \
-    mv /tmp/golangci-lint-${GOLANGCI_VERSION#v}-linux-amd64/golangci-lint /bin/golangci-lint
+# update: datasource=github-tags depName=kubernetes-sigs/kind
+ARG KIND_VERSION="v0.22.0"
+# update: datasource=github-tags depName=kubernetes/kubernetes
+ARG KUBECTL_VERSION="v1.27.3"
+# update: datasource=github-tags depName=kubernetes-sigs/kustomize extractVersion=^kustomize\/v(?<version>.+)$
+ARG KUSTOMIZE_VERSION="v5.3.0"
+# update: datasource=github-tags depName=aquasecurity/trivy
+ARG TRIVY_VERSION="v0.48.3"
```

Comment:
Why are we downgrading from 0.49.1?

Reply:
I updated it via this commit https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/commit/4acef3a204f4cde6c8a648b93d36251a897a7781 

thank you.